src/lib/account/account.ts

```diff
@@ -53,5 +54,16 @@ export const deriveAccountIdsAndMetadata: DeriveAccountIdsAndMetadata = async ar
       }),
     ),
   )
-  return merge({}, ...result)
+
+  const fulfilledAccountIdsAndMetadata = settledAccountIdsAndMetadata
+    .filter(result => {
+      if (isRejected(result)) {
+        console.error(result.reason)
+        return false
+      }
+      return isFulfilled(result)
+    })
+    .map(result => (result as PromiseFulfilledResult<AccountMetadataById>).value)
+
+  return merge({}, ...fulfilledAccountIdsAndMetadata)
```

Comment:
The guts of demo wallet being borked - we should handle rejections for accounts 0+ there or this will break the whole `<AppContext />` derivation / upsertion bit, resulting in accounts never being upserted and a seemingly infinite loading state for users